sig-docs/annual-report-2021.md

```diff
@@ -0,0 +1,128 @@
+What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+There is no official onboarding process. However, the SIG is supportive of new contributors,
+willing to walk through the contribution process and eager to see contributions completed.
+Issues are created regularly and triaged. Issues are also created specifically for new 
+contributors. There are issues labelled for the different contribution areas (web development, 
+blog, first time contributors, task, concept, API, etc) which helps new contributors 
+identify how to get started.
+
+From time to time we use the weekly meeting Zoom calls to help new contributors get onboard.
+
+**What programs do you participate in for new contributors?**
+
+We used to host doc sprints during the KubeCon,
+the doc sprints acted as a way to onboard new contributors as well as tackle a lot of open issues.
+
+**Does the group have contributors from multiple companies/affiliations?**
+
+Yes
+
+**Can end users/companies contribute in some way that they currently are not?**
+
+I suppose we're particular keen to hear about suggested improvements from people who are using Kubernetes in production and can see gaps in what's there.
+
+## Current initiatives and project health
+
+- [x] Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members.
+
+Longer term projects include
+
+- coordination with WG naming for things like removing the word “slave” (and other problematic terms) from docs.
+- publishing better information about releases kubernetes/website#20293
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? 
+Currently underway? What are some of the longer tail projects that your group is working on?**
+
+Shoutout to docsy theme work as well as the reference documentation generation :tada:.
+
+**Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+
+We have ongoing KEP, but we don't have a deadline set on when it should be finished.
+
+- [Doc policies for third party content](https://github.com/kubernetes/enhancements/tree/master/keps/sig-docs/1326-third-party-content-in-docs)
+  There's also an umbrella issue written as well [kubernetes/website#20232](https://github.com/kubernetes/website/issues/20232).
+
+
+**What areas and/or subprojects does the group need the most help with?**
+
+- Blog subproject
+
+**What's the average open days of a PR and Issue in your group? / what metrics does your group care about and/or measure?**
+
+We have the information from [devstat](https://k8s.devstats.cncf.io/d/25/open-pr-age-by-repository-group?orgId=1&var-period=q&var-repogroup_name=SIG%20Docs&var-kind_name=All):
+
+![Devstat 2020](devstats/devstats-quarter-2021.png)
+
+For example, based on 10/01/2020 data, we have:
+Average number of opened PRs 1.23 K with the median opened PRs age 3 days 13 hours 52 minutes 10 seconds.
```

Comment:
Maybe add that we informally review metrics quarterly. Also _ad hoc_ if anyone brings it up at a weekly meeting.

We aim to have < 100 open PRs for the English localization, and will take steps as needed if we see the figure climbing much above that.

Reply:
that's a nice goal / ballpark benchmark to call out